### ISSUE TITLE
Why different losses in train and test

### ISSUE THREAD

**Issue by helevor** [pankajmishra000/VT-ADL#6]
Hello, 

I can't understand why in train.py :
- loss = 5*loss1 + 0.5*loss2 + loss3 

and in test.py :
- in the function Thresholding : loss = loss1 + loss2 + loss3.sum()   (but it seems not used)
- in the function Patch_Overlap_Score : loss = loss1 -loss2 + loss3.max()     

In particular why the differences in the weights in comparison with the train loss ? (and why the "-" for the loss2 ?)

Could you please explain whe these differences or correct the code if it's a mistake ?

Thanks and regards

**Comment by pankajmishra000** [comment]
HI,
so first the code is correct and that's doing its intended function.

Here are their reason-
1- Training is done to capture the most important features and the best weights combinations are published for the used datasets.
2- Testing and post processing is one's choice and can be adapted based on the dataset and requirements. So thresholding value is found with the sum of all the losses and POS is found by adding the first two-loss and the max of the third loss. The negative sign is used because of the SSIM value.

**Comment by pankajmishra000** [comment]
i hope it solved your concern. thanks
